test/lit/basic/gc-atomics.wast

```diff
@@ -0,0 +1,149 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+
+;; RUN: wasm-opt -all %s -S -o - | filecheck %s
+;; RUN: wasm-opt -all %s --roundtrip -S -o - | filecheck %s
+
+(module
+  ;; CHECK:      (type $struct (struct (field (mut i32))))
+  (type $struct (struct (field (mut i32))))
+  ;; CHECK:      (type $packed (struct (field (mut i8))))
+  (type $packed (struct (field (mut i8))))
+
+  ;; CHECK:      (func $get (type $3) (param $0 (ref null $struct)) (result i32)
+  ;; CHECK-NEXT:  (struct.atomic.get seqcst $struct 0
```

Comment:
Is it maybe better not to print out the redundant `seqcst` here, to be more concise?